content/file-and-directory-naming/java/_index.md

```diff
@@ -12,10 +12,11 @@ title: File and directory naming in Java
 ---
 
 ## How should I name my Java files and directories?
-When you think about how to name files and directories in Java, you should know that different Java programmers can have different styles and approaches to the way they program. When naming stuff you must use Java standard conventions that are readable and meaningful to yourself and other programmers.
 
+When you think about how to name files and directories in Java, you should know that different Java programmers can have different styles and approaches to the way they program. When naming stuff you must use Java standard conventions that are readable and meaningful to yourself and other programmers.
```

Comment:
Is this not true for all languages? Do we already mention this in the general section?

All languages allow coders to be chaotic in how they name things
All languages have certain best practices and conventions around naming
All serious projects adopt a naming convention
All good programmers aim to be clear and consistent by making use of clear conventions instead of being chaotic

Another thing worth mentioning is that people have a bad habit of writing crazy horrible code, then trying to clean up the names after they get the code to work. This is seriously bad because it can introduce bugs, and the coder will often miss something. It's much better just to develop good habits from the start. It'll slow you down at first, but then you will move much faster when you get used to it.